css/theme-weebly.scss

```diff
@@ -16,46 +16,46 @@ $font-size-sm: 13px;
 $font-size-xs: 12px;
 
 $border-color: #dfe6ee;
-$border-radius: 6px;
+$border-radius: 2px;
 $border-radius-sm: 4px;
 
-$page-max-width: 972px;
+$page-max-width: 1024px;
 $sidebar-width: 220px;
 
 .nav-tabs > li.active > a,
 .nav-tabs > li.active > a:hover,
 .nav-tabs > li.active > a:focus {
-    background-color: #248ec2;
-    color: white !important;
+	background-color: #248ec2;
+	color: white !important;
 }
 
 div.navbar-collapse .dropdown-menu > li > a:hover {
-    background-color: #347DBE;
+	background-color: #347DBE;
 }
 
 a[data-toggle="tooltip"] {
```

Comment:
This file is still showing the switch from spaces to tabs. Let's keep it spaces. Since the entire file is red/green its hard to see the functional changes. 
![image](https://user-images.githubusercontent.com/51469/32121142-2f1f34aa-bb10-11e7-9273-0a86ca9475f4.png)